.github/workflows/on_build_do_test.yml

```diff
@@ -120,10 +119,17 @@ jobs:
             -Dorg.infinispan.test.server.extension.libs=org.jacoco:org.jacoco.agent:0.8.12:runtime
             -Dgithub.action=true
 
+      - uses: actions/create-github-app-token@v1
```

Comment:
Please add a comment explaining why you are creating a second token

Reply:
ok, done